### ISSUE TITLE
[Feature] GetVehicleBodies()?

### ISSUE THREAD

**Issue by Please-Pick-a-Name** [Teardown-Issue-Tracker-Maintainers/Teardown-Issue-Tracker#378]
### Is your feature request related to a problem (if not, explain why you think this feature should be added)? Please describe.

currently if one wants to find the all the vehicle body or its wheel body, they would have to loop through everything in the scene and check if this body belongs to any vehicle, its horrible

### Describe the solution you'd like

unlike GetVehicleBody(), this will return every vehicle body, include the main body, not-main body, and wheel body
no it dont need to worry about jointed bodies, one can still do GetJointedBodies()

### Describe alternatives you've considered

```lua
local bodies = FindBodies(nil,true)
for i = 1, #bodies do
    if GetBodyVehicle(bodies[i]) == vehicle then
        DrawBodyOutline(bodies[i]) -- asdfgh
    end
end
```

### Additional context

plz bless us

**Comment by Gregory-Gregory** [comment]
Hi @Please-Pick-a-Name, 

Thank you for this suggestion! It has been shared with the team.

**Comment by Please-Pick-a-Name** [comment]
> Hi @Please-Pick-a-Name,
> 
> Thank you for this suggestion! It has been shared with the team.

this is awkward, since this has already implemented now in 1.5
not sure why i forget to close this issue, please consider #495 instead
